spec/02-integration/04-admin_api/14-tags_spec.lua

```diff
@@ -0,0 +1,91 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson"
+
+-- We already test the functionality of page() when filtering by tag in 
```

Comment:
there are no tests for `/tags`?

* what happens with `/tags` with no arguments
* what happens with `/tags/valid_tag`
* what happens with `/tags/invalid@tag`
* what happens when mixing `/tags` and `?tags=`
  * what happens with `/tags/valid_tag?tags=another_tag`
  * what happens with `/tags/valid_tag?tags=another_tag,yet_another`
  * what happens with `/tags/valid_tag?tags=another_tag/yet_another`
  * what happens with `/tags/valid_tag?tags=invalid@tag`
* anything else?

Reply:
Added tests for /tags and /tags/:tags in spec/02-integration/04-admin_api/14-tags_spec.lua 👍